lib/Driver/Driver.cpp

```diff
@@ -672,6 +672,88 @@ getDriverBatchCount(llvm::opt::InputArgList &ArgList,
   return None;
 }
 
+static bool computeIncremental(const llvm::opt::InputArgList *ArgList,
+                               const bool ShowIncrementalBuildDecisions) {
+  {
+    const bool WasIncrementalRequested =
+    ArgList->hasArg(options::OPT_incremental);
+    if (!WasIncrementalRequested)
+      return false;
+  }
+  const char *ReasonToDisable =
+      ArgList->hasArg(options::OPT_whole_module_optimization)
+          ? "is not compatible with whole module optimization."
+          : ArgList->hasArg(options::OPT_embed_bitcode)
+                ? "is not currently compatible with embedding LLVM IR bitcode"
```

Comment:
Tiny nitpick: you're being inconsistent about periods here. (The old code used them in both messages.)

Reply:
Nice catch!